package.json

```diff
@@ -44,7 +44,8 @@
     },
     "scripts": {
         "test": "node ./bin/_lab -f -L -t 100 -m 10000 --types-test test/index.ts",
-        "test-cov-html": "node ./bin/_lab -f -L -r html -m 10000 -o coverage.html"
+        "test-cov-html": "node ./bin/_lab -f -L -r html -m 10000 -o coverage.html",
+        "lint": "node ./bin/_lab -d -f -L --lint-fix"
```

Comment:
Definitely remove `--lint-fix` if this gets removed. I recommend leaving lint script though.